component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -466,7 +466,8 @@ private void proceedWithOTP(HttpServletResponse response, AuthenticationContext
                 if (isUserExists) {
                     screenValue = getScreenAttribute(context, userRealm, tenantAwareUsername);
                     if (screenValue != null) {
-                        url = url + SMSOTPConstants.SCREEN_VALUE + screenValue;
+                        url = url + SMSOTPConstants.SCREEN_VALUE + screenValue + "&order="+ SMSOTPUtils
```

Comment:
missing space near "&order="+

Reply:
Fixed in 4066f62025c6f56790f00d24add5fa86c0a4a810